tests/manifests/test-cluster-on-pvc-encrypted.yaml

```diff
@@ -14,7 +14,7 @@ spec:
           requests:
             storage: 5Gi
   cephVersion:
-    image: quay.io/ceph/ceph:v17.2.6
+    image: quay.io/ceph/daemon-base:latest-reef-devel
```

Comment:
I think we should have some tag around this and some automated script to update the image on all the tests. WDYT?

Reply:
Some canary tests run with v18, some v17.2.6, and the golang tests run with [these or other devel images](https://github.com/rook/rook/blob/master/tests/framework/installer/ceph_installer.go#L45-L52). The issue in this case is that we need the canary tests to run in our daily CI against devel images, and not just with the PRs against those released images.